Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,54 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+default persistent._mas_pm_swear_frequency = None
```

Comment:
I'd make constants for the corresponding values for swear frequency (`SF_OFTEN` (2), `SF_SOMETIMES` (1), `SF_NEVER` (0))

Would make things easier to use should we ever need to otherwise.

Reply:
Thanks, I've added this.